src/vanna/pgvector/pgvector.py

```diff
@@ -251,15 +251,3 @@ def remove_collection(self, collection_name: str) -> bool:
 
     def generate_embedding(self, *args, **kwargs):
         pass
-
-    def submit_prompt(self, *args, **kwargs):
```

Comment:
If you remove these naively, you will run into this:
```
E       TypeError: Can't instantiate abstract class PG_VectorStore with abstract methods assistant_message, submit_prompt, system_message, user_message
```

So more code changes are required if you do not want these.

Reply:
I was copying https://github.com/vanna-ai/vanna/blob/main/src/vanna/vannadb/vannadb_vector.py

I need this change, otherwise I get a different error as the response is `None`.  But I know what you mean, this seems to be a feature of inheriting from `VannaBase` for both vector stores and LLM models.

Reply:
Looking into it I think the solution is to import the two classes the other way around

```
class CustomVanna(OpenAI_Chat, PG_VectorStore):
    def __init__(self, config=None):
        OpenAI_Chat.__init__(self, config=config)
        PG_VectorStore.__init__(self, config=config)
```

So I can potentially revert this change, although I was basing the import order on the [docs](https://vanna.ai/docs/postgres-openai-standard-qdrant/)

Reply:
I found the issue with the default embedding model. I will make a PR to your branch.

Reply:
I made a PR to your branch now:
https://github.com/edlouth/vanna/pull/1

I have also made changes to the unit test, which I will make a separate PR to your branch for.

Reply:
Thanks for sorting that. I have merged the PR on my branch, so it is in this PR now.

Reply:
I made another PR now, which reverts the methods you removed again, as they are not needed.
I also updated the PGVector unit tests:
https://github.com/edlouth/vanna/pull/2

Reply:
I made another PR now, which reverts the methods you removed again, as they are not needed.
I also updated the PGVector unit tests:
https://github.com/edlouth/vanna/pull/2